src/Windows/Avalonia.Win32/Win32Platform.cs

```diff
@@ -115,7 +116,8 @@ public static void Initialize(Win32PlatformOptions options)
                 .Bind<PlatformHotkeyConfiguration>().ToSingleton<PlatformHotkeyConfiguration>()
                 .Bind<IPlatformIconLoader>().ToConstant(s_instance)
                 .Bind<NonPumpingLockHelper.IHelperImpl>().ToConstant(new NonPumpingSyncContext.HelperImpl())
-                .Bind<IMountedVolumeInfoProvider>().ToConstant(new WindowsMountedVolumeInfoProvider());
+                .Bind<IMountedVolumeInfoProvider>().ToConstant(new WindowsMountedVolumeInfoProvider())
+                .Bind<IPlatformColorSchemeProvider>().ToConstant(new WindowsColorSchemeProvider(!options.UseSystemAccentColor));
```

Comment:
I think it's easier to not register WindowsColorSchemeProvider, if options.UseSystemAccentColor is false.

Reply:
mmm, u right